x-pack/plugins/data_enhanced/server/routes/session.ts

```diff
@@ -49,6 +49,7 @@ export function registerSessionRoutes(router: IRouter): void {
           body: response,
         });
       } catch (err) {
+        logger.error(err);
```

Comment:
I added these logs since, if the error was an actual Internal Server Error then no useful information would be carried through in the response. The only place to see a stack trace of the error would be here.